programs/clearing_house/src/controller/orders.rs

```diff
@@ -496,6 +496,15 @@ pub fn fill_order(
     controller::funding::settle_funding_payment(user, &user_key, &mut market, now)?;
     controller::lp::settle_lp(user, &user_key, &mut market, now)?;
 
+    validate!(
```

Comment:
need this in fill_spot_order?

Reply:
should we block place_order?

Reply:
theres no risk w/ place order and its nice if users can place while fills are paused?